lib/util/http/jira.ts

```diff
@@ -0,0 +1,22 @@
+import type { HttpOptions, HttpResponse, InternalHttpOptions } from './types';
+import { Http } from '.';
+
+let baseUrl = 'https://api.atlassian.com/';
```

Comment:
Yes, at the moment for the larger PR functionality to work, setBaseURL() must be called.  

There are two identical sets of API collections to integrate with [Jira Cloud REST API v3](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#authentication)
* `https://<site-url>/rest/api/3/<resource-name>`
* `https://api.atlassian.com/ex/jira/<cloudId>/rest/api/3/<resource-name>`

I flipped back and forth between the two for ages, but decided on using the `site-url` version since it's more commonly known by users of Jira Cloud.  CloudID is not a readily accessible field.

That said, both attributes can be returned from the Bitbucket Cloud APIs...

Perhaps it's better to not have a default baseURL value, like that in the `gitea` or `bitbucket-server`, which forces an explicit setBaseUrl().  Thoughts about this approach?

Reply:
yes, should have no default value